docs/administration.rst

```diff
@@ -390,6 +390,36 @@ This notification type also supports the same ``--exclude`` and ``EXCLUDE_CN_FRO
 NOTE: At present, this summary email essentially duplicates the certificate expiration notifications, since all
 certificate expiration notifications are also sent to the security team. This issue will be fixed in the future.
 
+**Notification configuration**
+
+The following configuration options are supported:
+
+.. data:: EXCLUDE_CN_FROM_NOTIFICATION
+    :noindex:
+
+    Specifies CNs to exclude from notifications. This includes both single notifications as well as the notification summary. The specified exclude pattern will match if found anywhere in the certificate name.
+
+    .. note::
+        This is only used for celery. The equivalent for cron is '-e' or '--exclude'.
+
+       ::
+
+          DISABLE_NOTIFICATION_PLUGINS = ['exclude', 'also exclude']
+
+
+.. data:: DISABLE_NOTIFICATION_PLUGINS
+    :noindex:
+
+    Specifies a set of notification plugins to disable. Notifications will not be sent using these plugins. Currently only applies to expiration notifications, since they are the only type that utilize plugins.
```

Comment:
shall we add that this could be off value in the test environment, where you would want to trigger and test the notification job, but not actually send out the notification.

Reply:
Good idea, added!